src/app/create-project/create-project.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="new-project-form">
+  <h2> Create a new project</h2>
+  <form>
+    <div class="form-group">
+      <label for="name" class="form-label">Project Name</label>
+      <input #projectName required minlength="3" type="text" class="form-control" ngModel name="name" #name="ngModel"
+             id="name" placeholder="My New Project">
+    </div>
+    <div class="form-group">
+      <label for="api" class="form-label">API base url</label>
+      <input #api type="url" class="form-control" id="api" placeholder="/isAlive">
+    </div>
+    <div class="form-group">
+      <label for="general" class="form-label">General Description</label>
+      <input #desc type="text" class="form-control" id="general"
+             placeholder="Checks if the url sends a ping back">
+    </div>
+    <div class="form-group">
+      <button
+        *ngIf="!(name.invalid && (name.dirty || name.touched) && (name.errors.minlength || name.errors.required)) && projectName.value.length !== 0"
```

Comment:
"ngIf"s shouldn't be so long, if you need conditions that are long, move this to a function